### PR TITLE
Assorted Changes, Layouts mostly

### PR DESCRIPTION
**Pull request by andrewsilke**
Icon tweaks, combo box added functionality.
Needs to be merged to match stylesheets update.

### CODE REVIEW

File: zoo/libs/tooldata/tooldata.py [hunk 1]
```diff
@@ -175,7 +175,14 @@ def __getattr__(self, item):
     def __setattr__(self, key, value):
         self[key] = value
 
-    def save(self):
+    def save(self, dump=False):
```
Comment (dsparrow27) [inline zoo/libs/tooldata/tooldata.py]: better to call it 'indent' over 'dump'

Comment (andrewsilke) [reply zoo/libs/tooldata/tooldata.py]: yep sounds good

Comment (andrewsilke) [reply zoo/libs/tooldata/tooldata.py]: done

**Comment by andrewsilke** [comment]
should be good to go now

**Pull request merged**